serverless/aws/provider.py

```diff
@@ -24,6 +24,8 @@ class Runtime(yaml.YAMLObject):
     PYTHON_3_7 = "python3.7"
     PYTHON_3_8 = "python3.8"
     PYTHON_3_9 = "python3.9"
+    PYTHON_3_10 = "python3.10"
```

Comment:
It would be also good to update `node` runtime accordingly to currently supported https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html we can also remove old python ones that are not supported anymore

Reply:
Updated accordingly